### PR TITLE
conky: add Spanish translation

### PR DESCRIPTION
**Pull request by ivanhercaz**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/linux/conky.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`conky`
+
+- Crear una nueva configuración por defecto:
+
+`conky -C > ~/.conkyrc`
+
+-Ejecuta conky con un archivo de configuración concreto:
+
+`conky -c {{path/to/config}}`
```
Comment (sbrl) [inline pages.es/linux/conky.md]: Is this in Spanish?

Comment (zdroid) [reply pages.es/linux/conky.md]: No, it definitely isn't. :laughing: It's something like `camino/a/la/config`.

Comment (ivanhercaz) [reply pages.es/linux/conky.md]: > Is this in Spanish?

Haha, no, it isn't Spanish. It was a little slip I had translating.

> No, it definitely isn't.  It's something like `camino/a/la/config`.

Yes, it is something similar. It should be changed to `{{ruta/a/la/configuración}}`.

Comment (ivanhercaz) [reply pages.es/linux/conky.md]: edbcb9d solved this review, @sbrl, @zdroid. Thank you for point this.

**Pull request merged**